src/libraries/System.Runtime.InteropServices.WindowsRuntime/src/System.Runtime.InteropServices.WindowsRuntime.csproj

```diff
@@ -4,6 +4,9 @@
     <IsPartialFacadeAssembly>true</IsPartialFacadeAssembly>
     <TargetFrameworks>$(NetCoreAppCurrent)-Windows_NT;$(NetCoreAppCurrent)</TargetFrameworks>
     <Nullable>enable</Nullable>
+    <GenFacadesIgnoreMissingTypes Condition="'$(RuntimeFlavor)' == 'Mono'">true</GenFacadesIgnoreMissingTypes>
+    <GenFacadesIgnoreMissingType Condition="'$(RuntimeFlavor)' == 'Mono'">true</GenFacadesIgnoreMissingType>
```

Comment:
is there a reason why we have both `GenFacadesIgnoreMissingTypes` and `GenFacadesIgnoreMissingType` here?

Reply:
Because both were necessary to fix this 😄 

The errors here, and what we're tracking, is still a bit new too me so I took the simple approach of suppressing them all. I'm going to expand this out to the full type list here and file and issue to follow up on the gap. That should make it clearer why both are necessary (assuming they still are).

Reply:
@akoeplinger I already commented on that above. `GenFacadesIgnoreMissingTypes` specifies that the facades should be built at all. `GenFacadesIgnoreMissingType` should be items inside `<ItemGroup>` that specify the full names of the missing types.